client/components/search/index.jsx

```diff
@@ -99,7 +100,7 @@ class Search extends Component {
 	constructor( props ) {
 		super( props );
 
-		this.instanceId = uniqueId();
+		this.instanceId = uuid();
```

Comment:
There is a `withInstanceId` helper in `@wordpress/compose` that could be used to replace this.

Reply:
Agreed. But we'll still a more standalone and non-component version for instances in state or library functions (some are in this PR). So I prefer this one for now unless you feel strongly about using `withInstanceId`.